src/Tests/TestComponentCSharp/Class.cpp

```diff
@@ -1480,6 +1480,25 @@ namespace winrt::TestComponentCSharp::implementation
         return winrt::make<service_provider>();
     }
 
+    struct __declspec(uuid("15651B9F-6C6B-4CC0-944C-C7D7B0F36F81")) IComInterop : ::IUnknown
```

Comment:
Can we add a test similar to this model that exercises one of the interfaces that we added ref const for?  I see this one is consumed via a ComImport in the test and it would be useful for us to custom implement one of the other interfaces and verify we can call it.  This would allow for a test that runs in the test pass which exercises it.

Reply:
yes, implemented IDragDropManager interop interface.  note that this required InternalsVisibleTo from Windows to UnitTest, which in turn created an ambiguity in WinrtModule in the WinUI test (which was doing nothing), so I removed that line